src/shared/components/palette/dynamic-radio-list/write-dynamic-radio-list-field.html

```diff
@@ -0,0 +1,20 @@
+<div class="form-group bottom-30"
+    [ngClass]="{'form-group-error': !dynamicRadioListControl.valid && dynamicRadioListControl.touched}" [id]="id()">
+  <fieldset>
+    <label [for]="id()">
+        <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
+        <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+        <span class="form-label" *ngIf="!caseField.label && !caseField.hint_text">Select an option below</span>
+        <span class="error-message"
+            *ngIf="dynamicRadioListControl.errors && dynamicRadioListControl.touched">{{dynamicRadioListControl.errors |
```

Comment:
Pretty sure this should be using the `.dirty` property, not `.touched`.

Reply:
Agreed, update to reflect .dirty

Reply:
updated to dirty.